general/development/process/triage.md

```diff
@@ -0,0 +1,233 @@
+
+Users may also comment with additional details, screenshots, replicating instructions. It may happen that the issue gets re-evaluated and priority or summary changed.
+
+### Revisiting old issues
+
+While searching the tracker you may come over issues that were reported long time ago but still remain open. Again, ask the following questions:
+
+- **Is this still an issue?** If it is not applicable any more either close it or comment about it on the issue and recommend to close. Very few tracker users actually have permission to close issues but the component lead or watchers will see your comment and revisit issue. Another good practice is to replace `triaged` label with `triaging_in_progress` and add a comment asking if the issue can be closed.
+
+:::warning Closing issues
+
+If users confirm that it was resolved or nobody replies in 30 days, issue should be closed.
+
+:::
+
+- **Do affected versions need correction?** If the issue is still applicable, make sure to add missing current affected versions or comment about it on the issue if you can't edit it.
+- **Does the issue have patch and if yes, is it still applicable?** If the issue has patch that still works but `patch` label is missing, look through comments or history to see if the `patch` label was removed after reviewing the current patch. If you find that label was never added, do it yourself. On contrary if the issue has `patch` label but the patch is no longer applicable or not sufficient, remove the label.
+- **Are there any duplicating issues?** When finding duplicates among the old issues it might not be obvious which issue to close as a duplicate. Usually we should leave the first reported issue but if the later issues have more watchers, better description, more votes, useful comments, attached patch, etc. you may decide to close the earlier issue and leave the later. Sometimes both issues have lots of watchers and they both remain open.
+
+:::tip
+
+In any case, always create links between duplicates or related issues.
+
+:::
+
+- **Does the issue have assignee who forgot about it or misleading status `Development in process`?** Due to some process changes in 2013 some issues still have a real user in the **Assignee** field but this user actually does not work on the issue. Sometimes **Assignee** remains filled after failing `Peer review`, sometimes developers simply forget that the issue was assigned to them. If you suspect that **Assignee** does not actually work on the issue, comment asking they about it and in some cases remove the **Assignee** completely. Allow somebody else to work on the issue without feeling that the issue is "taken". Please also note that for some time tracker had a restriction that **Assignee** could not be empty so you can find lots of issues assigned to **moodle.com** or **Nobody**. Do not modify such issues as this creates unnecessary activity, emails to watchers and irrelevant change in the "Last update date".
+
+### Comments templates
+
```

Comment:
I'd suggest adding a tip admonition to tell people to check out the Canned responses.

I'd almost be tempted to suggest we entirely remove these and only use Canned responses beacuse these will likely become stale and out-of-date over time.

Reply:
Hi Andrew! 
I had the same thought while reviewing this page so your comment has convinced me to remove these template messages from this page and replace them with a reference to canned responses in the tracker :-)